pages/common/jmeter.md

```diff
@@ -0,0 +1,20 @@
+# jmeter
+
+> Open source java application designed for load testing functional behavior and measure performance.
+> More information: <https://jmeter.apache.org>.
+
+- Run jmeter in cli mode for a test plan:
+
+`jmeter -n -t {{name}}.jmx`
+    
+- Run jmeter and specify the log output:
+
+`jmeter -n -t {{name}}.jmx -l {{logfile}}.jtl`
+
+- Run jmeter using a proxy server and a port:
+
+`jmeter -n -t {{name}}.jmx -H {{proxy_server_url}} -P {{port}}`
```

Comment:
It's better to provide a sample value instead of `proxy_server_url`:

```suggestion
`jmeter --nongui --testfile {{path/to/file}}.jmx --proxyHost {{proxy_server_url}} --proxyPort {{port}}`
```

```suggestion
`jmeter --nongui --testfile {{path/to/file.jmx}} --proxyHost {{proxy_server_url}} --proxyPort {{port}}`
```

Reply:
I was not too sure if I should specify or not.
Please validate ✅